packages/builder/src/components/design/PropertiesPanel/PropertyControls/FilterEditor/FilterDrawer.svelte

```diff
@@ -39,17 +45,30 @@
     ]
   }
 
-  const removeFilter = id => {
+  function getSchemaFields(table) {
+    const base = table ? Object.values(table.schema) : []
+    // if internal table, can use the _id field as well
+    const isInternal = table && !table.sql
+    if (isInternal && !base.find(field => field.name === "_id")) {
+      base.push({ name: "_id", type: "string" })
+    }
+    if (isInternal && !base.find(field => field.name === "_rev")) {
+      base.push({ name: "_rev", type: "string" })
+    }
+    return base
+  }
+
+  function removeFilter(id) {
     filters = filters.filter(field => field.id !== id)
   }
 
-  const duplicateFilter = id => {
+  function duplicateFilter(id) {
```

Comment:
We have an endless battle of updating functions in svelte components to be either arrow functions or normal functions lol.